### PR TITLE
Add two commands to aid in cleaning a dirty build.

### PR DESCRIPTION
**Pull request by jasongrout**
I'm often running these commands manually to make sure I'm starting fresh.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -61,7 +61,9 @@
     "build:css:base": "lessc --strict-math=on --include-path=node_modules src/basestyle/index.less lib/basestyle/index.css",
     "build:css:theme": "lessc --strict-math=on --include-path=node_modules src/default-theme/index.less lib/default-theme/index.css",
     "clean": "rimraf docs && rimraf lib && rimraf test/build && rimraf test/coverage",
+    "clean:all": "npm run clean && cd jupyterlab && npm run clean && cd ..",
     "clean:examples": "node scripts/cleanexamples.js",
+    "reset": "npm run clean:all && rimraf jupyterlab/node_modules && rimraf node_modules && npm install && npm run build:all",
```
Comment (sccolbert) [inline package.json]: s/reset/rebuild ?

Comment (jasongrout) [inline package.json]: Sure, I'll make the change.

Comment (jasongrout) [inline package.json]: On second thought, then it's confusing what the difference between `build` and `rebuild` is. I'd prefer to take out the build step and call it `reset` or something else that indicates that it gives you a clean slate.

Comment (jasongrout) [inline package.json]: How about that? `clean:slate`?

File: scripts/buildexamples.js [hunk 2]
```diff
@@ -7,14 +7,7 @@ var fs = require('fs');
...
 
-var cmd;
 for (var i = 0; i < dirs.length; i++) {
-  if (dirs[i].indexOf('.') !== -1) {
-    continue;
-  }
-  if (dirs[i].indexOf('node_modules') !== -1) {
-    continue;
-  }
   console.log('Building: ' + dirs[i] + '...');
```
Comment (blink1073) [inline scripts/buildexamples.js]: I've been meaning to add some better delimiters here (like a line of asterisks), want to do that in this PR?

**Comment by sccolbert** [comment]
:shipit:

**Pull request merged**